### PR TITLE
Disable stdout logging when --quiet is used.

### PR DESCRIPTION
**Pull request by markstory**
This changes stdout/stderr logging to better match the CLI flags being used. Now `--quiet` turns off all stdout logging, while `--verbose` enables debug logging.

I felt this setup took the spirit of #7417 and improved upon it. I'm targetting 3.1 as I don't see adjusting the logging levels of CLI tools as a breaking or notable change.

### CODE REVIEW

**Comment by ravage84** [comment]
Looks reasonable. :cookie: 

Migration guide needs to be updated.
Probably we should also note the new behavior under http://book.cakephp.org/3.0/en/console-and-shells.html#console-output-levels

**Comment by markstory** [comment]
@ravage84 I can update the documentation if/when this is merged.

File: src/Console/Shell.php [hunk 1]
```diff
@@ -457,6 +450,27 @@ public function runCommand($argv, $autoMethod = false, $extra = [])
...
+        $this->_io->setLoggers(ConsoleIo::NORMAL);
+        if (!empty($this->params['quiet'])) {
+            $this->_io->level(ConsoleIo::QUIET);
+            $this->_io->setLoggers(ConsoleIo::QUIET);
+        }
+        if (!empty($this->params['verbose'])) {
+            $this->_io->level(ConsoleIo::VERBOSE);
+            $this->_io->setLoggers(ConsoleIo::VERBOSE);
+        }
+    }
```
Comment (lorenzo) [inline src/Console/Shell.php]: I always wanted to extract this to another method :)

**Comment by lorenzo** [comment]
Great job :+1:

**Pull request merged**